test/e2e/codecept.conf.js

```diff
@@ -82,7 +71,7 @@ exports.config = {
   },
   plugins: {
     stepByStepReport: {
-      enabled: true,
```

Comment:
Why is this set to False...?Any particular reason...?

Reply:
This plugin records each page within the journey, adds to longer test runs and wouldn't be best practice for CI/CD